jenkins-centos7-atomicapp/README.md

```diff
@@ -0,0 +1,102 @@
+This is an atomicapp application based on the nulecule specification. Docker and Kubernetes are the supported providers. You'll need to run this from a workstation that has the [atomic](https://github.com/projectatomic/atomic) command and kubectl client that can connect to a kubernetes master.
+
+It is a single tier application based on jenkins.
+
+### Option 1: Interactive
+
+Run the image. It will automatically use kubernetes as the orchestration provider.  It will prompt for all parameters in the Nulecule file that do not have default values:
+
+    $ [sudo] atomic run projectatomic/jenkins-centos7-atomicapp
+
+## Option 2: Unattended
+
+1. Create the file `answers.conf` with these contents:
+
+    This sets up the values for the configurable parameters and indicates that kubernetes should be the orchestration provider.
+
+        [general]
+        namespace = default
+        provider = kubernetes
+
+        [jenkins]
+        image=kumarpraveen/centos-jenkins:latest
+        # port on the node where Jenkins over HTTP will be accessible - valid range 30000-32667
+        NODE_PORT=30000
```

Comment:
Earlier in the sample answer file, you have mentioned `NODE_PORT`=30001, use similar port (even if giving sample).